test/services/ElasticsearchRepositoryTest.java

```diff
@@ -42,26 +41,6 @@ public void testAddAndQueryResources() throws IOException {
   }
 
   @Test
-  public void testAddAndEsQueryResources() throws IOException {
```

Comment:
Obsolete: because all of its functionality is also tested by BaseRepositoryTest.
Failing: because there is no link count when putting Resources to ES directly (without the functionality of BaseRepositoryTest).